modules/rtm/tests/testthat/test.resample.R

```diff
@@ -0,0 +1,51 @@
+library(testthat)
+library(PEcAnRTM)
+context("Resampling functions")
+
+test_that(
+  "Univariate resampling works",
+  {
+    from <- seq(2, 20, 2)
+    to <- seq(2, 20)
+    values <- from ^ 2
+    true_values <- to ^ 2
+    resample_values <- resample(values, from, to)
+    expect_equal(true_values, resample_values)
+  }
+)
+
+test_that(
+  "Multivariate resampling works",
+  {
+    m <- matrix(1:20, ncol = 2)
+    true_m <- cbind(seq(1, 10, 0.5), seq(11, 20, 0.5))
+    resample_m <- resample(m, 1:10, seq(1, 10, 0.5))
+    expect_equal(true_m, resample_m)
```

Comment:
@ashiklom is this so you can resample the spec res of a matrix (i.e. image)?

Reply:
An image would probably be a cube, which this doesn't quite cover (I suppose I could try to extend this to arrays with 3+ dimensions, but it's a bit more complicated), but generally, yeah, the idea is if you have multiple observations, all at the same spectral resolution, you can pass them directly into `resample` without having to manually `apply` or loop and `cbind`.